tools/setup/emoji_dump/emoji_dump.py

```diff
@@ -8,18 +8,22 @@
 import sys
 import xml.etree.ElementTree as ET
 from six import unichr
+from os.path import abspath, dirname, join
 
 from PIL import Image, ImageDraw, ImageFont
 
+ZULIP_PATH = dirname(dirname(dirname(dirname(abspath(__file__)))))
```

Comment:
for this I'd write os.path.join(abspath(__file__), "../../../../") -- it's a lot shorter